components/Button/react.js

```diff
@@ -6,7 +6,7 @@ import s from './style.module.scss'
 const cx = classNames.bind(s)
 
 const Button = ({
-  children, disabled, fullWidth, onClick, variant, size, uppercase,
+  children, disabled, fullWidth, onClick, variant, size, uppercase, ...props
```

Comment:
This is pretty cool, but if we're not acting on a prop being passed in, what would be a use case for spreading everything to button, if we already have all the valid props destructured?

Reply:
Ah, this must be for the aria stuff 👌

Reply:
@tarikhamilton Correct. I would rather be explicit for destructuring props however spread seemed like the best option for the scope of the PR.